pyro/ops/einsum/torch_map.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import, division, print_function
+
+import operator
+
+from six.moves import reduce
+
+from pyro.ops import packed
+from pyro.ops.einsum.adjoint import Backward, einsum_backward_sample, transpose, unflatten
+from pyro.ops.einsum.util import Tensordot
+
+assert transpose  # pacify flake8
```

Comment:
nit: We have used `# noqa F403` to pacify flake8's import not used error in other modules. Can we use the same idiom?

Reply:
I dislike the noqa F403 for two reasons:
1. it is ambiguous in that only one of the imports is unused here (`transpose`), and `noqa` allows other extraneous imports on the same `import` line
2. on long import lines that are wrapped, the `#noqa f403` can end up appearing on only one line, indeed in this PR appeared on a line different from the `transpose` it was intended to block

But I'll concede if you feel strongly that `f403` is a better solution 😄

Reply:
Nevermind, it looks like `__all__ = ...` works fine. Thanks for suggesting!

Reply:
I don't feel strongly at all, and your reasons are all valid, but it will be nice to maintain consistency whichever way it is. Also, you might not even need it (I think flake8 picks up `__all__` usage e.g. in `distributions.__init__`).